config.js

```diff
@@ -34,6 +34,12 @@ module.exports = {
   // See `bufferSize`.
   flushDelaySeconds: 30,
 
+  // If true, traces will be published as soon as `bufferSize` traces have
+  // been collected if this occurs before `flushDelaySeconds`.
+  // If false, traces will flush every `flushDelaySeconds` and additional
+  // traces beyond `bufferSize` will be dropped.
+  publishImmediately: true,
+
```

Comment:
how about `samplingEnabled`? That captures the idea that traces will be lost and the description above it gives precise semantics.